app/metrics.yaml

```diff
@@ -130,3 +130,130 @@ settings_screen:
     notification_emails:
       - android-probes@mozilla.com
     expires: "2022-08-17"
+
+tracking_protection:
+  toolbar_shield_clicked:
+    type: counter
+    description: |
+      A counter that indicates how many times a user has opened
+      the tracking protection settings panel from the toolbar.
+    bugs:
+      - https://github.com/mozilla-mobile/focus-android/issues/5057
+    data_reviews:
+      - https://github.com/mozilla-mobile/focus-android/pull/5163
+    data_sensitivity:
+      - interaction
+    notification_emails:
+      - android-probes@mozilla.com
+    expires: "2022-08-17"
+
+  tracking_protection_changed:
+    type: event
+    description: |
+      The user has changed the setting for enhanced tracking protection.
+    bugs:
+      - https://github.com/mozilla-mobile/focus-android/issues/5057
+    data_reviews:
+      - https://github.com/mozilla-mobile/focus-android/pull/5163
+    data_sensitivity:
+      - interaction
+    notification_emails:
+      - android-probes@mozilla.com
+    expires: "2022-08-17"
+
+  has_ever_changed_etp:
```

Comment:
Per my other comment, you can make this a [`lifetime: user`](https://mozilla.github.io/glean/book/user/metrics/adding-new-metrics.html#user) and you wouldn't have to set it with every execution, just once and Glean would remember the value for you.